test/validation/org/apache/cassandra/HarnessTest.java

```diff
@@ -180,7 +180,7 @@ static URL getConfigURL(String yamlPath)
         }
     }
 
-    public String getTestName(String yamlPath)
+    static String getTestName(String yamlPath)
```

Comment:
yeah.. it got nothing to do with this PR.. I just noticed that while I was working on another function and just made it static.